packages/core/admin/admin/src/content-manager/components/BlocksEditor/BlocksInput/index.js

```diff
@@ -48,12 +50,36 @@ const BlocksInput = ({ readOnly }) => {
   const blocks = useBlocksStore();
   const renderElement = React.useCallback((props) => baseRenderElement(props, blocks), [blocks]);
 
+  const handleEnter = () => {
+    const selectedNode = editor.children[editor.selection.anchor.path[0]];
```

Comment:
What is `anchor.path`? I logged the value but I don't really understand. The code reads really nicely but I feel this part could maybe use a comment or variable name to clarify...?

Reply:
Yes I know it's not intuitive, I had to ask chatgpt the other day 😁

In this case anchor and focus points should be the same because it's just about finding out where the cursor is, I don't think a range of text should be selected.

<img width="526" alt="Screenshot 2023-09-21 at 11 30 35" src="https://github.com/strapi/strapi/assets/8087692/d30b542d-60a2-4134-8748-e64b1eb592b6">